### ISSUE TITLE
Check and update stored objects in kube-apiserver

### ISSUE THREAD

**Issue by ymmt2005** [cybozu-go/neco#904]
## What

Check if there are objects stored in etcd that are having deprecated API versions in etcd.
Update them by empty-patching, if any.

## How

1. Check Kubernetes 1.17 Release note for deprecated API versions
2. Dump cke-etcd and check if there are objects having deprecated versions
3. Convert them by empty-patching.

## Checklist

- [x] Finish implementation of the issue
- [x] Test all functions
- [x] Have enough logs to trace activities
- [x] Notify developers of necessary actions

**Comment by tapih** [comment]
I checked `API Changes` and `Deprecations and Removals` on the [CHANGELOG](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.17.md), and found the following GV is deprecated.
 - `rbac.authorization.k8s.io/v1alpha1`
 - `rbac.authorization.k8s.io/v1beta1`

Then, I executed the following command and found that nothing should be changed.
```
@stage0-boot-0
ckecli etcd root-issue --output file
env ETCDCTL_API=3 etcdctl --cacert=./etcd-ca.crt --cert=./etcd-root.crt --key=./etcd-root.key --endpoints=https://10.69.0.21:2379 get --prefix /registry | grep -a -v last-applied | grep -a "rbac.authorization.k8s.io/v1beta1"
```

In addition to that, `kubeadm.k8s.io/v1beta1` is also deprecated.
I searched the GV from the top of github.com/cybozu-go, and found that nothing should be changed.

So, in short, nothing needs to be changed here. Just close.
